tensorflow-cpu.yaml

```diff
@@ -0,0 +1,126 @@
+#nolint:valid-pipeline-git-checkout-tag
+package:
+  name: tensorflow-cpu
+  version: 0_git20250108
+  epoch: 0
+  description: tensorflow-cpu
+  copyright:
+    - license: Apache-2.0
+  target-architecture:
+    - x86_64
+  dependencies:
+    runtime:
+      - bash
+      - build-base
+      - curl
+      - pkgconf
+      - py${{vars.py-version}}-pip
+      - python-${{vars.py-version}}-dev
```

Comment:
Is it intentional to ship the dev variant of Python?

Reply:
Yep, I just stuck to what upstream does ([here](https://github.com/tensorflow/build/blob/35a7dbc754193523e5582ce5b5cbbb4c07cff7b2/tensorflow_runtime_dockerfiles/setup.python.sh#L31)). Do you think it's only needed at build time for the next pip-installs?

Reply:
@maxgio92 I'm honestly unsure. Could we test with the non-dev variant?

Reply:
Sure @EyeCantCU. Tests and build seem to pass even without that 🤔

Reply:
Let's send it and come back if problematic